util/blooms-db/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "blooms-db"
+version = "0.1.0"
+authors = ["debris <marek.kotewicz@gmail.com>"]
```

Comment:
This should be `admin@parity.io`?

Reply:
😅

Reply:
Forgot one thing,  also add a license I guess it should be: `license = "GPL-3.0"`

Reply:
I believe it doesn't matter for subcrates :)

Reply:
actually, you are right. We usually have a `license` field so I'll also add it :)